default/stringtables/en.txt

```diff
@@ -10448,91 +10448,92 @@ SHP_WEAPON_1_2
 Mass Driver 2
 
 SHP_WEAPON_1_2_DESC
-Improves Mass Driver by one level, increasing strength by 1.
+Upgrades [[shippart SR_WEAPON_1_1]] parts on all ships within [[metertype METER_SUPPLY]], increasing shot [[encyclopedia DAMAGE_TITLE]] to 4.
 
 SHP_WEAPON_1_3
 Mass Driver 3
 
 SHP_WEAPON_1_3_DESC
-Improves Mass Driver by two levels, increasing strength by 2.
+Upgrades [[shippart SR_WEAPON_1_1]] parts on all ships within [[metertype METER_SUPPLY]], increasing shot [[encyclopedia DAMAGE_TITLE]] to 5.
 
 SHP_WEAPON_1_4
 Mass Driver 4
 
 SHP_WEAPON_1_4_DESC
-Improves Mass Driver by three levels, increasing strength by 3.
+The ultimate refinement of the Mass Driver. Upgrades [[shippart SR_WEAPON_1_1]] parts on all ships within [[metertype METER_SUPPLY]], increasing shot [[encyclopedia DAMAGE_TITLE]] to 6.
```

Comment:
Further improvements of (at least) mass drivers have been discussed, might drop the use of `ultimate` or any hints at a final refinement.